chromadb/api/models/Collection.py

```diff
@@ -49,11 +49,10 @@ def __init__(
         else:
             import chromadb.utils.embedding_functions as ef
 
-
             logger.warning(
-                "No embedding_function provided, using default embedding function: SentenceTransformerEmbeddingFunction"
+                "No embedding_function provided, using default embedding function: DefaultEmbeddingFunction"
```

Comment:
Just a suggestion... saying "the default embedding function is DefaultEmbeddingFunction" isn't very informative, should we signal what's actually being used somehow?

Reply:
Also, you still get this message which is kind of confusing in use cases where I'm supplying my own embedding vectors and not using any embedding function at all.

Any way we could defer this warning until we actually *use* the embedding function?

Reply:
Added a link to the model being used. As for the deference. I will leave that change to another PR since I'm trying to not alter behavior but just swap the model here. I think @atroyn wanted to make the model a default param and had some thoughts there.